wpilibj/src/athena/java/edu/wpi/first/wpilibj/I2C.java

```diff
@@ -18,72 +17,74 @@
  */
 public class I2C extends SensorBase {
   public enum Port {
     kOnboard(0), kMXP(1);
-    private int value;
+    private int m_value;
 
-    private Port(int value) {
-      this.value = value;
+    Port(int value) {
+      this.m_value = value;
     }
 
     public int getValue() {
-      return this.value;
+      return this.m_value;
     }
-  };
+  }
 
-  private Port m_port;
-  private int m_deviceAddress;
+
+  private final Port m_port;
+  private final int m_deviceAddress;
 
   /**
    * Constructor.
    *
-   * @param port The I2C port the device is connected to.
+   * @param port          The I2C port the device is connected to.
    * @param deviceAddress The address of the device on the I2C bus.
    */
   public I2C(Port port, int deviceAddress) {
-    m_port = port;
-    m_deviceAddress = deviceAddress;
+    this.m_port = port;
+    this.m_deviceAddress = deviceAddress;
 
-    I2CJNI.i2CInitialize((byte) m_port.getValue());
+    I2CJNI.i2CInitialize((byte) port.getValue());
 
     UsageReporting.report(tResourceType.kResourceType_I2C, deviceAddress);
   }
 
   /**
    * Destructor.
    */
-  public void free() {}
+  public void free() {
+  }
 
   /**
    * Generic transaction.
    *
-   * This is a lower-level interface to the I2C hardware giving you more control
-   * over each transaction.
+   * <p>This is a lower-level interface to the I2C hardware giving you more control over each
+   * transaction.
    *
-   * @param dataToSend Buffer of data to send as part of the transaction.
-   * @param sendSize Number of bytes to send as part of the transaction.
+   * @param dataToSend   Buffer of data to send as part of the transaction.
+   * @param sendSize     Number of bytes to send as part of the transaction.
    * @param dataReceived Buffer to read data into.
-   * @param receiveSize Number of bytes to read from the device.
+   * @param receiveSize  Number of bytes to read from the device.
    * @return Transfer Aborted... false for success, true for aborted.
    */
-  public synchronized boolean transaction(byte[] dataToSend, int sendSize, byte[] dataReceived,
-      int receiveSize) {
-    int status = -1;
+  public synchronized boolean transaction(byte[] dataToSend, int sendSize,
+                                          byte[] dataReceived, int receiveSize) {
+    final int status;
 
     ByteBuffer dataToSendBuffer = ByteBuffer.allocateDirect(sendSize);
     if (sendSize > 0 && dataToSend != null) {
       dataToSendBuffer.put(dataToSend);
     }
     ByteBuffer dataReceivedBuffer = ByteBuffer.allocateDirect(receiveSize);
 
-    status =
-        I2CJNI.i2CTransaction((byte) m_port.getValue(), (byte) m_deviceAddress, dataToSendBuffer,
-            (byte) sendSize, dataReceivedBuffer, (byte) receiveSize);
+    status = I2CJNI
+        .i2CTransaction((byte) m_port.getValue(), (byte) m_deviceAddress,
+            dataToSendBuffer, (byte) sendSize, dataReceivedBuffer,
+            (byte) receiveSize);
```

Comment:
Kind of a strange reformat.  I personally find this harder to read than the earlier version.